builder/layouts/snippet.hbs

```diff
@@ -10,24 +9,25 @@
             height: 100%;
         }
     </style>
-    <meta name="twitter:card" content="summary_large_image" />
+    <meta name="twitter:card" content="summary_large_image"/>
     <meta name="twitter:site" content="@30sec_angular">
     <meta name="twitter:title" content="✨ 30 Seconds of Angular: {{title}} ✨">
     <meta name="twitter:text:title" content="✨ 30 Seconds of Angular ✨">
     <meta name="twitter:description" content="{{title}}">
     <meta name="twitter:image" content="{{site.base}}{{slugify title}}/preview.png">
     <meta name="twitter:image:alt" content="30 seconds of angular">
-{{/head }}
+    <title>✨ {{title}} - {{site.title}}</title>
+{{/head}}
 
-{{#> full-body }}
+{{#> body}}
 
-{{!-- don't indent, it'll break code examples --}}
-{{>snippet}}
+{{!--
+This comment prevent indenting snippets which breaks code examples.
+--}}{{>snippet}}
 
-    {{#if (hasCode this)}}
-        <div class="mdl-card mdl-shadow--2dp card demo-card">
-            <iframe src="https://codelab.fun/angular/playground?code={{codeToBase64 this}}"></iframe>
-        </div>
-    {{/if}}
-
-{{/full-body}}
+{{#if (hasCode this)}}
```

Comment:
indentation?

Reply:
I think comment above screwed it up, it's actually how it is formatted. 

We might get prettier or something.